x-pack/plugins/entity_manager/server/lib/entities/built_in/containers.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EntityDefinition, entityDefinitionSchema } from '@kbn/entities-schema';
+import { BUILT_IN_ID_PREFIX } from './constants';
+
+export const builtInContainersFromEcsEntityDefinition: EntityDefinition =
+  entityDefinitionSchema.parse({
+    id: `${BUILT_IN_ID_PREFIX}containers_from_ecs_data`,
+    managed: true,
+    version: '1.0.0',
+    name: 'Containers from ECS data',
+    description:
+      'This definition extracts container entities from common data streams by looking for the ECS field container.id',
+    type: 'container',
+    indexPatterns: ['filebeat-*', 'logs-*', 'metrics-*', 'metricbeat-*'],
+    identityFields: ['container.id'],
+    displayNameTemplate: '{{container.id}}',
```

Comment:
This should be `container.name`.

Reply:
Please see https://github.com/elastic/elastic-entity-model/issues/168